pkg/volume/azure_file/azure_file.go

```diff
@@ -289,7 +291,15 @@ func (b *azureFileMounter) SetUpAt(dir string, mounterArgs volume.MounterArgs) e
 		mountOptions = appendDefaultMountOptions(mountOptions, mounterArgs.FsGroup)
 	}
 
-	err = b.mounter.Mount(source, dir, "cifs", mountOptions)
+	mountComplete := false
+	err = wait.Poll(5*time.Second, 10*time.Minute, func() (bool, error) {
+		err := b.mounter.Mount(source, dir, "cifs", mountOptions)
+		mountComplete = true
+		return true, err
+	})
+	if !mountComplete {
```

Comment:
do we need to kill the mount process if it has been hung?

Reply:
point is how to kill it in the mount cifs code?
now if it's timeout, then it would return error, and then try mount again.
This is happening on an edge case, not expected to happen frequently.

Reply:
is it possible to return the process id on mount error? it may need to refactor mount codes I think

Reply:
`mounter.Mount` func don't have that info, may need refactor mount codes

Reply:
I see. Let's add the timeout first